tests/udf_communication/peer_communication/test_add_peer.py

```diff
@@ -28,7 +28,7 @@
     logger_factory=WriteLoggerFactory(file=Path(__file__).with_suffix(".log").open("wt")),
     processors=[
         structlog.contextvars.merge_contextvars,
-        ConditionalMethodDropper(method_name="debug"),
+        #ConditionalMethodDropper(method_name="debug"),
```

Comment:
```suggestion
        # ConditionalMethodDropper(method_name="debug"), 
```

Leftover?

Reply:
ups, actually needs to be uncommented, because otherwise the logs are too large and things too slow, it filter debug level logs out